docs/content/en/docs-dev/user-guide/managing-application/configuration-drift-detection.md

```diff
@@ -50,7 +50,7 @@ This status means the application is deploying and the configuration drift detec
 
 ### How to enable
 
-This feature is automatically enabled for all applications.
+By default, drift detection is **NOT** enabled. You can enable it for each application by configuring application's [OnOutOfSync.disabled](../configuration-reference/#onoutofsync).
```

Comment:
The "drift detection" feature is the detection feature, not the reconcile feature.
The option you mention in this PR is the reconcile option; the deployment will be triggered when the sync state is OUT_OF_SYNC.

So, I mean, the drift detection is enabled by default.

Reply:
Yes, the drift detection is enabled by default. The disabled code is disabled the auto trigger new deployment on OUT_OF_SYNC by default @t-kikuc

Reply:
thanks both, 
yeah, drift detection is enabled by default, and deployment on OutOfSync is not.

I recovered the previous sentence and added a new explanation. [2f0785b](https://github.com/pipe-cd/pipecd/pull/5120/commits/2f0785b8dc0082e8c6c5339d602d0b32cf39fb4b)

```
This feature is automatically enabled for all applications.

If you want to deploy automatically when `OUT_OF_SYNC` occures, configure each application's [OnOutOfSync](../configuration-reference/#onoutofsync).
```

Reply:
@t-kikuc There is a long-time story about this spec 😓 
ref: https://github.com/pipe-cd/pipecd/discussions/2752
tl;dr: Previously, we didn't support trigger deployment on OUT_OF_SYNC, which someone thinks that it didn't suit the GitOps spec, so we decided to add that feature. The problem is, if we make that trigger by default, our users at that time will be affected (due to a ton of unexpected deployment triggers whenever OUT_OF_SYNC happens). That's why we make this spec - disable trigger deployment on OUT_OF_SYNC by default. 😄